tracer/src/Datadog.Trace/Span.cs

```diff
@@ -4,6 +4,7 @@
 // </copyright>
 
 using System;
+using System.Collections.Generic;
```

Comment:
nit: presumably these are accidental additions?
```suggestion
```

Reply:
Fixed in 937873794